documentation/modules/configuring/proc-scaling-down-a-kafka-cluster.adoc

```diff
@@ -1,36 +1,47 @@
 // Module included in the following assemblies:
 //
-// ???
+// configuring/assembly-scaling-clusters.adoc
 
 [id='proc-scaling-down-a-kafka-cluster-{context}']
 
 = Scaling down a Kafka cluster
 
-This procedure describes how to decrease the number of brokers in a Kafka cluster.
+[role="_abstract"]
+Use a reassignment file to decrease the number of brokers in a Kafka cluster.
+
+The reassignment file must describe how partitions are reassigned to the remaining brokers in the Kafka cluster.
+Brokers in the highest numbered pods are removed first.
```

Comment:
Will it still be the case when we remove stateful sets?

Reply:
Yes, it will.